[BUG] context.go:12:25: undefined: c.MaxScale build for loong64 arch on arm64 machine

### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Current Behavior

Build loong64 arch failed on arm64 machine


### Expected Behavior

Successfully

### Steps To Reproduce

1. Run 'GOOS=linux GOARCH=loong64 make'


### Relevant log output

```shell
rm -f './bin/nuclei' 2>/dev/null
CGO_ENABLED=0 go build -trimpath -v -pgo=auto -ldflags '-s -w -extldflags "-static"'  \
                 -o './bin/nuclei' cmd/nuclei/main.go
github.com/ericlagergren/decimal
# github.com/ericlagergren/decimal
/Users/go/pkg/mod/github.com/ericlagergren/decimal@v0.0.0-20221120152707-495c53812d05/context.go:12:25: undefined: c.MaxScale
make: *** [go-build] Error 1
```

### Environment

```markdown
- OS: Apple M1 Pro MacOS 15.6.1 (24G90)
- Nuclei: 3.6.0
- Go: 1.25.5
```

### Anything else?

_No response_

--- a/go.mod
+++ b/go.mod
@@ -213,7 +213,7 @@ require (
 	github.com/dsnet/compress v0.0.2-0.20230904184137-39efe44ab707 // indirect
 	github.com/ebitengine/purego v0.8.4 // indirect
 	github.com/emirpasic/gods v1.18.1 // indirect
-	github.com/ericlagergren/decimal v0.0.0-20221120152707-495c53812d05 // indirect
+	github.com/ericlagergren/decimal v0.0.0-20240411145413-00de7ca16731 // indirect
 	github.com/fatih/color v1.18.0 // indirect
 	github.com/felixge/fgprof v0.9.5 // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect

